src/GraphQL.Tests/Bugs/Bug138DecimalPrecisionTests.cs

```diff
@@ -1,11 +1,16 @@
 using GraphQL.Types;
+using Shouldly;
 using Xunit;
 
 namespace GraphQL.Tests.Bugs
 {
     public class Bug138DecimalPrecisionTests : QueryTestBase<DecimalSchema>
     {
+#if NETCOREAPP3_1
         [Fact]
+#else
+        [Fact(Skip = "24.149999999999999 with .NET Core < 3.1")]
```

Comment:
@Shane32 Earlier this test was also ignored. Then I deleted #if/#else. I didn't understand how it worked before. Now I saw that the test started to fail and added again #if/#else. Also I see weird messages from windows build:
```
D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL\GraphQL.csproj : error NU1101: Unable to find package Microsoft.SourceLink.GitHub. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL.sln]
D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL\GraphQL.csproj : error NU1102: Unable to find package NETStandard.Library with version (>= 2.0.3) [D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL.sln]
D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL\GraphQL.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 1.6.0 ] [D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL.sln]
D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL\GraphQL.csproj : error NU1101: Unable to find package GraphQL-Parser. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL.sln]
D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL\GraphQL.csproj : error NU1102: Unable to find package Microsoft.CSharp with version (>= 4.7.0) [D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL.sln]
D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL\GraphQL.csproj : error NU1102:   - Found 2 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.1 ] [D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL.sln]
D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL\GraphQL.csproj : error NU1102: Unable to find package System.ComponentModel.Annotations with version (>= 4.7.0) [D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL.sln]
D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL\GraphQL.csproj : error NU1102:   - Found 2 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.1.0 ] [D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL.sln]
D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL\GraphQL.csproj : error NU1101: Unable to find package System.Reactive.Core. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL.sln]
D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL\GraphQL.csproj : error NU1101: Unable to find package System.Reactive.Linq. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [D:\a\graphql-dotnet\graphql-dotnet\src\GraphQL.sln]
```

I've seen this before but thought it was a single glitch. Some info I found: https://github.com/actions/virtual-environments/issues/1090#issuecomment-748432535 I don’t have time to go into more detail yet. If you want then go on